packages/storybook/stories/components/input/3_demo.stories.ts

```diff
@@ -27,16 +26,8 @@ const storyParams = {
   },
   defaultValue: {
     category: 'General',
-    control: { type: 'text' },
-  },
-  value: {
-    category: 'General',
-    control: { type: 'text' },
-  },
-  icon: {
-    category: 'General',
-    defaultValue: '',
-    options: ODS_ICON_NAMES,
+    defaultValue: 'md',
```

Comment:
<img width="210" alt="Capture d’écran 2023-12-22 à 10 23 16" src="https://github.com/ovh/design-system/assets/53512103/f45b1367-42de-44d4-aa87-04cf78c9f20d">
Something went wrong in this file. 
Size is no longer displayed as control in Input story so, it should be removed here.
And we should still be able to play with value and icon in the controls.